packages/react/src/Tooltip/Tooltip.module.css

```diff
@@ -0,0 +1,156 @@
+.Tooltip {
+  display: none;
+}
+
+@media (forced-colors: active) {
+  .Tooltip {
+    outline: 1px solid transparent;
+  }
+}
+
+/* Overriding the default popover styles */
+.Tooltip[popover] {
+  /* these vars are for the triangle, via https://www.smashingmagazine.com/2024/03/modern-css-tooltips-speech-bubbles-part1/ */
+  --b: 1em; /* triangle base width */
+  --h: 0.5em; /* triangle height */
+  --p: 50%; /* center triangle position (0%:left 100%:right) */
+  --r: var(--brand-borderRadius-medium); /* border radius */
+  --c: var(--brand-color-neutral-emphasisPlus); /* bg color */
+
+  position: absolute;
+  padding: 0.5em 0.75em;
+  width: max-content;
+  margin: auto;
+  clip: auto;
+  white-space: normal;
+  font: normal normal var(--brand-text-size-100) / var(--brand-text-lineHeight-100) var(--brand-fontStack-sansSerif);
+  -webkit-font-smoothing: subpixel-antialiased;
+  color: var(--brand-color-text-onEmphasis);
+  text-align: center;
+  word-wrap: break-word;
+  background: var(--c);
+  border-radius: var(--r);
+  border: 0;
+  opacity: 0;
+  max-width: 250px;
+  inset: auto;
+  /* for scrollbar */
+  overflow: visible;
+}
+
+.Tooltip[popover]:popover-open, /* selector in chrome is :popover-open */
+.Tooltip[popover].\:popover-open {
+  /* selector in firefox and safari is \:popover-open */
+  display: block;
+  animation-name: tooltip-appear;
+  animation-duration: 0.1s;
```

Comment:
The fastest animation timing in Brand so far is 0.2s... could that work here?

![Screenshot 2024-03-25 at 16 34 49](https://github.com/primer/brand/assets/13340707/cf6655b2-59d2-48b4-aa78-98e28c05d978)